mod_admin_audit.erl

```diff
@@ -53,6 +55,9 @@ observe_search_query(#search_query{search={audit, Args}}, Context) ->
 observe_search_query(#search_query{}, _Context) ->
     undefined.
 
+observe_tick_1h(tick_1h, Context) ->
```

Comment:
1h seems like a high frequency here. maybe daily would be better?

Reply:
One hour is not that high of a frequency. 

Also in the beginning it will have to get rid of millions of audit records. It will drop them in batches of 10000, so it needs a high enough frequency to do this. With this frequency it will take a couple of days to gradually remove the records.